pkg/operator/controller/certificate-publisher/controller.go

```diff
@@ -106,12 +112,20 @@ func (r *reconciler) secretToIngressController(o client.Object) []reconcile.Requ
 		listOpts = client.MatchingFields(map[string]string{
 			"defaultCertificateName": o.GetName(),
 		})
+		ingressConfig configv1.Ingress
 	)
 	if err := r.cache.List(context.Background(), &list, listOpts); err != nil {
 		log.Error(err, "failed to list ingresscontrollers for secret", "secret", o.GetName())
 		return requests
 	}
+	if err := r.cache.Get(context.Background(), controller.IngressClusterConfigName(), &ingressConfig); err != nil {
+		log.Error(err, "failed to get ingress 'cluster'")
```

Comment:
Nit: Use `controller.IngressClusterConfigName()` in lieu of the hard-coded 'cluster'?

Reply:
Updated.